db/src/Pos/DB/Block.hs

```diff
@@ -46,54 +43,34 @@ import           System.IO.Error (IOError, isDoesNotExistError)
 
 import           Pos.Binary.Class (decodeFull', serialize')
 import           Pos.Block.BHelpers ()
-import           Pos.Block.Types (Blund, SlogUndo (..), Undo (..))
+import           Pos.Block.Types (SlogUndo (..), Undo (..))
 import           Pos.Core (HeaderHash, headerHash)
 import           Pos.Core.Block (Block, GenesisBlock)
 import qualified Pos.Core.Block as CB
 import           Pos.Crypto (hashHexF)
 import           Pos.DB.BlockIndex (deleteHeaderIndex, putHeadersIndex)
 import           Pos.DB.Class (MonadDB (..), MonadDBRead (..), Serialized (..),
                      SerializedBlock, SerializedBlund, SerializedUndo,
-                     getBlock, getDeserialized)
+                     getBlock)
 import           Pos.DB.Error (DBError (..))
-import           Pos.DB.GState.Common (getTipSomething)
 import           Pos.DB.Pure (DBPureVar, MonadPureDB, atomicModifyIORefPure,
                      pureBlocksStorage)
 import           Pos.DB.Rocks (MonadRealDB, blockDataDir, getNodeDBs)
 import           Pos.DB.Sum (MonadDBSum, eitherDB)
 import           Pos.Delegation.Types (DlgUndo (..))
 import           Pos.Util.Util (HasLens (..), eitherToThrow)
 
-----------------------------------------------------------------------------
--- BlockDB related methods
-----------------------------------------------------------------------------
-
-getUndo :: MonadDBRead m => HeaderHash -> m (Maybe Undo)
-getUndo = getDeserialized dbGetSerUndo
-
--- | Convenient wrapper which combines 'dbGetBlock' and 'dbGetUndo' to
--- read 'Blund'.
---
--- TODO Rewrite to use a single call
-getBlund :: MonadDBRead m => HeaderHash -> m (Maybe (Block, Undo))
-getBlund x =
-    runMaybeT $
-    (,) <$> MaybeT (getBlock x)
-        <*> MaybeT (getUndo x)
-
--- | Store blunds into a single file.
---
---   Notice that this uses an unusual encoding, in order to be able to fetch
---   either the block or the undo independently without re-encoding.
-putBlunds :: MonadDB m => NonEmpty Blund -> m ()
-putBlunds = dbPutSerBlunds
-          . map (\bu@(b,_) -> ( CB.getBlockHeader b
-                              , Serialized . serialize' $ bimap serialize' serialize' bu)
-                )
-
--- | Get 'Block' corresponding to tip.
-getTipBlock :: MonadDBRead m => m Block
-getTipBlock = getTipSomething "block" getBlock
```

Comment:
These had to move to another module in the same package to avoid an import loop caused by the fact that there already was a `Pos.DB.Block` module, but we also wanted that module to re-export the stuff below.